lib/robots/dor_repo/gis_delivery/load_vector.rb

```diff
@@ -42,50 +32,34 @@ def perform_work
             # sniff out shapefile from extraction
             Dir.chdir(tmpdir)
             shpfn = Dir.glob('*.shp').first
```

Comment:
removed the do/end block around the `Dir.chdir` else we had issues with the `sqlfn` being scoped to the block but being required below this outside the block by the "load data into PostGIS" command